src/SDK/Services/DeclineReasonsService.cs

```diff
@@ -0,0 +1,106 @@
+                    "Could not create decline reasons for account." + " Exception: " + e.Message, e.ServerError, e);
+            }
+            catch (OssException e)
+            {
+                throw  new OssException("Could not create decline reasons for account." + " Exception: " + e.Message, e);
+            }
+        }
+        
+        public IList<string> UpdateDeclineReasons(IList<string> declineReasons, string language)
+        {
+            string path = template.UrlFor(UrlTemplate.DECLINE_REASONS_PATH).Replace("{language}", language)
+                .Build();
+
+            try
+            {
+                string serializeObject = JsonConvert.SerializeObject(declineReasons);
+                string stringResponse = restClient.Put(path, serializeObject);
+                return JsonConvert.DeserializeObject<IList<string>>(stringResponse, settings);
+            }
+            catch (OssServerException e)
+            {
+                throw new OssServerException(
+                    "Could not update decline reasons for account." + " Exception: " + e.Message, e.ServerError, e);
+            }
+            catch (OssException e)
+            {
+                throw  new OssException("Could not update decline reasons for account." + " Exception: " + e.Message, e);
+            }
+        }
+
+        public IList<string> GetDeclineReasons(string language)
+        {
+            string path = template.UrlFor(UrlTemplate.DECLINE_REASONS_PATH).Replace("{language}", language)
+                .Build();
+
+            try
+            {
+                string stringResponse = restClient.Get(path);
+                return JsonConvert.DeserializeObject<IList<string>>(stringResponse, settings);
+            }
+            catch (OssServerException e)
+           {
+               throw new OssServerException(
+                   "Could not get decline reasons for account." + " Exception: " + e.Message, e.ServerError, e);
+           }
+           catch (OssException e)
+           {
+               throw  new OssException("Could not get decline reasons for account." + " Exception: " + e.Message, e);
+           }
+        }
+        
+        public void DeleteDeclineReasons(string language)
+        {
+            string path = template.UrlFor(UrlTemplate.DECLINE_REASONS_PATH).Replace("{language}", language)
+                .Build();
+
+            try
+            {
+                string stringResponse = restClient.Delete(path);
+            }
+            catch (OssServerException e)
+            {
+                throw new OssServerException(
+                    "Could not get decline reasons for account." + " Exception: " + e.Message, e.ServerError, e);
+            }
+            catch (OssException e)
+            {
+                throw  new OssException("Could not get decline reasons for account." + " Exception: " + e.Message, e);
+            }
+        }
+        
+    }
+}
```

Comment:
This service is not required because AccountConfigService will be used. Please delete it.